spring-petclinic-microservices/docker-compose.yml

```diff
@@ -69,15 +69,6 @@ services:
     ports:
      - 8080:8080
 
-  tracing-server:
```

Comment:
do we have a task to track adding the tracing back?

Reply:
https://github.com/Azure/azure-sdk-for-java/issues/26306